packages/core/src/lib/wallet/connect.ts

```diff
@@ -6,6 +6,7 @@ declare global {
     coin98?: {
       keplr: KeplrWindow;
     };
+    fin: WalletWindowInterface
```

Comment:
Can you make this attribute optional?

`fin?: WalletWindowInterface`

Reply:
Yes, this code is fixed at latest commit. Thanks for your review